### PR TITLE
Fix/mobile button position

### PR DESCRIPTION
**Pull request by santiagonoguez**
Conditionally change the height depending if a mobile browser is found or not.

Tested on both desktop and mobile browser:
https://apps-stage-9.risevision.com/templates/edit/d4a5a9c7-2cc7-4f99-87cf-1e8457d0ea9a?cid=cf85e5c4-9439-40ce-94d6-e5ea6ea2411c

common-header PR: https://github.com/Rise-Vision/common-header/pull/938
As usual, I will update the reference to it when it's approved.

### CODE REVIEW

File: web/scripts/template-editor/controllers/ctr-template-editor.js [hunk 1]
```diff
@@ -9,6 +9,12 @@ angular.module('risevision.template-editor.controllers')
       $scope.isTestCompanySelected = userState.isTestCompanySelected;
       $scope.hasUnsavedChanges = false;
 
+      $scope.isMobileBrowser = function() {
+        return /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(
```
Comment (stulees) [inline web/scripts/template-editor/controllers/ctr-template-editor.js]: @santiagonoguez Do you know if we've provided this type of function elsewhere in Apps? If we haven't, this feels like it's more appropriate to be a reusable utility function. Thoughts?

Comment (santiagonoguez) [reply web/scripts/template-editor/controllers/ctr-template-editor.js]: @stulees I searched but couldn't find something in that project. 

I think it's a good idea, but where that utility function would be located ?

@fjvallarino

Comment (fjvallarino) [reply web/scripts/template-editor/controllers/ctr-template-editor.js]: @santiagonoguez @stulees we don't really have a global `utils` service. We do have `svc-presentation-utils.js`, were we put some logic common to both Editors

Comment (santiagonoguez) [reply web/scripts/template-editor/controllers/ctr-template-editor.js]: @fjvallarino thanks ! I'll use that !

File: web/scripts/editor/services/svc-presentation-utils.js [hunk 2]
```diff
@@ -3,10 +3,16 @@
...
   .constant('HTML_PRESENTATION_TYPE', 'HTML Template')
   .factory('presentationUtils', ['HTML_TEMPLATE_TYPE', 'HTML_PRESENTATION_TYPE',
-    '$state', 'checkTemplateAccess', 'plansFactory',
-    function (HTML_TEMPLATE_TYPE, HTML_PRESENTATION_TYPE, $state, checkTemplateAccess, plansFactory) {
+    '$state', '$window', 'checkTemplateAccess', 'plansFactory',
+    function (HTML_TEMPLATE_TYPE, HTML_PRESENTATION_TYPE, $state, $window, checkTemplateAccess, plansFactory) {
       var factory = {};
 
+      factory.isMobileBrowser = function() {
+        return /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(
```
Comment (stulees) [inline web/scripts/editor/services/svc-presentation-utils.js]: @santiagonoguez Curious, is this a list you sourced somewhere?

Comment (santiagonoguez) [reply web/scripts/editor/services/svc-presentation-utils.js]: @stulees yes, it's the top answer here ( not depending on jquery ):

https://stackoverflow.com/questions/3514784/what-is-the-best-way-to-detect-a-mobile-device-in-jquery

**Pull request merged**